src/components/button/Button.jsx

```diff
@@ -0,0 +1,17 @@
+// Style
+import './button.css';
+
+const Button = ({ type, value, onClick, disabled, text }) => {
+    return (
+        <button
+            type={type}
+            value = {value ? value : null}
+            onClick={onClick}
+            disabled = {disabled ? disabled : null}
```

Comment:
Als het goed is werkt dit ook gewoon
```suggestion
            disabled = {disabled}
```